### PR TITLE
docs: improve readme

### PR DESCRIPTION
**Pull request by trym-b**
Best reviewed commit by commit

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,3 +35,13 @@ The actual contents depends on the project you have.
 
 If you have 3rd party dependencies, their licenses should be placed in
 `/licenses-dependencies/<name-of-dependency>`
+
+### Set up dependabot
+
+Create a new label Github label `dependencies` by following
```
Comment (maeb) [inline README.md:41]:
```suggestion
Create a new Github label `dependencies` by following
```

Comment (trym-b) [reply README.md:41]: There can never be enough labels labels labels!

Will fix

File: README.md [hunk 2]
```diff
@@ -10,13 +10,19 @@ This repo exists to more easily create a container based service with `golang`
...
+
 ### Set up branch protection rules
 
 The default (`main`) branch should be protected, so please ensure that the
 branch is sufficiently protected. The following should be enabled:
 
 * Require minimum number of PR reviewers
-* Require `Test` and `Lint` workflow jobs to pass before merging
+* Require `Test`, `Lint`, `CodeQl build` and `CodeQL`(from Github Code
+  Scanning), workflow jobs to pass before merging
```
Comment (maeb) [inline README.md:25]:
```suggestion
  Scanning) workflow jobs to pass before merging
```

**Review by maeb — APPROVED** [review]
LGTM

**Pull request merged**